### PR TITLE
Pokemon card buttons

### LINKED ISSUE #98: Pokemon Card buy and sell button
Animate buy, sell and value button when hovering over a pokemon card.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
_List Changes Introduced by this PR_
1. Added buttons on pokemon cards for the value, sell and buy options. 
2. Added animation on buttons. 

## Purpose
Users will be able to buy pokemon cards while browsing in the pokedex.

Closes #98

### CODE REVIEW

File: pokeapp/src/styling/components/_pokemon-card.scss [hunk 1]
```diff
@@ -126,16 +126,94 @@
...
+    0% {
+        transform: translateX(0%);
+        opacity: 1;
+    }
+
+    100% {
+        transform: translateX(-100%);
+        opacity: 0;
+    }
+}
```
Comment (JorgeLVilla) [inline pokeapp/src/styling/components/_pokemon-card.scss:219]: Ooooh. Excited to see this in action.

**Review by Lap343 — APPROVED** [review]
Oh very nice. Was trying to wrap my head around the code but you have it to where the card zooms in once. Nice job

**Review by JorgeLVilla — APPROVED** [review]
Nice job Jay!

**Comment by JayDampitan** [comment]
Thanks fam!

**Pull request merged**